be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -314,9 +314,13 @@ Status ScrollParser::fill_tuple(const TupleDescriptor* tuple_desc, Tuple* tuple,
             continue;
         }
 
-        tuple->set_not_null(slot_desc->null_indicator_offset());
         const rapidjson::Value& col = line[col_name];
+        if ((pure_doc_value && col.IsArray() && col[0].IsNull()) || col.IsNull()) {
```

Comment:
Can you give this document example for this condition. As far as I know, doc_value would not be null